app/helpers/works_helper.rb

```diff
@@ -96,12 +96,12 @@ def marked_for_later?(work)
     reading && reading.toread?
   end
 
-  def marktoread_link(work)
-    link_to ts("Mark for Later"), marktoread_work_path(work)
+  def markasread_link(work)
```

Comment:
This should be `mark_as_read_link(work)` now that we're using the underscores between each word.